ebml_matroska.xml

```diff
@@ -862,19 +862,66 @@
     <documentation lang="en">Contains the command information. The data SHOULD be interpreted depending on the ChapProcessCodecID value. <a href="https://www.matroska.org/technical/specs/chapters/index.html#dvd">For ChapProcessCodecID = 1</a>, the data correspond to the binary DVD cell pre/post commands.</documentation>
   </element>
   <element name="Tags" path="0*(\Segment\Tags)" id="0x1254C367" type="master" minver="1" webm="0">
-    <documentation lang="en">Element containing Elements specific to Tracks/Chapters. A list of valid tags can be found <a href="https://www.matroska.org/technical/specs/tagging/index.html">here.</a></documentation>
+    <documentation lang="en">Element containing metadata describing Tracks, Editions, Chapters, Attachments, or the Segment as a whole. A list of valid tags can be found <a href="https://www.matroska.org/technical/specs/tagging/index.html">here.</a></documentation>
   </element>
   <element name="Tag" path="1*(\Segment\Tags\Tag)" id="0x7373" type="master" minOccurs="1" minver="1" webm="0">
-    <documentation lang="en">Element containing Elements specific to Tracks/Chapters.</documentation>
+    <documentation lang="en">A single metadata descriptor.</documentation>
   </element>
   <element name="Targets" path="1*1(\Segment\Tags\Tag\Targets)" cppname="TagTargets" id="0x63C0" type="master" minOccurs="1" maxOccurs="1" minver="1" webm="0">
-    <documentation lang="en">Contain all UIDs where the specified meta data apply. It is empty to describe everything in the Segment.</documentation>
+    <documentation lang="en">Contain all UIDs that relate the specified metadata of the Tag to what the Tag is described. If empty or not present, then the Tag describes everything in the Segment.</documentation>
```

Comment:
"…to what the Tag is described" makes no sense to me (as a non-native English speaker). As a long-time Matroska developer I know what `Targets` does, but I image someone unfamiliar with Matroska will be very confused when reading this.

Alternative suggestions: "Contains all UIDs that this tag applies to." or "Specifies which other elements the metadata represented by the Tag applies to."